FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo/ReadMe.md

```diff
@@ -46,21 +46,46 @@ the location of source files:
 You can changes these directory to let the proyejct work with a different
 source tree.
 
-The Multiple Interface Visual studio demo showcases the Multiple Interfaces (or
-rather the multiple endpoints) functionality of the FreeRTOS+TCP
-IPv6/multi-interface library. The Windows Simulator environment doesn't actually
-have multiple interfaces which can be used by FreeRTOS and thus, this demo shows
+## Introduction
+
+The IPv6_Multi_WinSim Visual studio demo showcases the Multiple Interfaces (or
+rather the multiple endpoints) functionality of the FreeRTOS+TCP library devIPv6
```

Comment:
Should we mention the exact branch name: `dev/IPv6_integration` if the user checks that out from the +TCP library.

Reply:
devIPv6 is the FreeRTOS branch, You are suggesting to add TCP branch as well?

Reply:
> FreeRTOS+TCP library branch.

seems like we are specifying the branch name of the library (FreeRTOS+TCP) rather than FreeRTOS

Reply:
Sure, let me update it to avoid the confusion.